### PR TITLE
Integration test database, debugging assistance

### PR DESCRIPTION
**Pull request by patimen**
## Integration tests results upload to BigQuery
As they finish, we upload the Integration test results to a GCP BigQuery database for future analysis (by humans and AI)

## Debugging messages improved
Trying to diagnose some issues in integration test failures

## Fuzz testing
An experiment in fuzz testing yielded several edge cases.

### CODE REVIEW

File: decentralized-api/internal/validation/inference_validation.go [hunk 1]
```diff
@@ -1105,7 +1113,7 @@ func positionDistance(
 	validationLogprobs []completionapi.TopLogprobs,
 ) (float64, error) {
 	if len(originalLogprobs) == 0 || len(validationLogprobs) == 0 {
-		return 0.0, fmt.Errorf("empty logprobs provided")
+		return 0.0, nil
```
Comment (DimaOrekhovPS) [inline decentralized-api/internal/validation/inference_validation.go]: Why isn't it an error now?

**Review by DimaOrekhovPS — APPROVED** [review]

File: decentralized-api/internal/validation/inference_validation.go (hunk 1 above)
Comment (patimen) [reply decentralized-api/internal/validation/inference_validation.go]: Looking at the logic above this, I think you're right, I'll revert.
